tests/features/api.feature

```diff
@@ -0,0 +1,320 @@
+        | Room     | B-28 03/A150                                                                               |
+
+      And the following Press Contact entity:
+        | Name               | A press contact                                                                          |
+        | Address            | country_code: HU - locality: Szeged - address_line1: Press contact 1 - postal_code: 6700 |
+        | Email              | press@example.com                                                                        |
+        | Phone number       | +32477777778                                                                             |
+        | Social media links | uri: http://facebook.com - title: Facebook - link_type: facebook                         |
+
+      And the following General Contact entity:
+        | Name               | A general contact                                                                            |
+        | Address            | country_code: BG - locality: Varna - address_line1: General contact 1 - postal_code: 9009    |
+        | Email              | general@example.com                                                                          |
+        | Phone number       | +359525566778                                                                                |
+        | Social media links | uri: http://instagram.com - title: Instagram - link_type: instagram                          |
+
+      And the Event Content "My event" is updated as follows:
+        | Venue                 | DIGIT                               |
+        | Contact               | A press contact, A general contact  |
+
+      When I visit node "My event" edit page
+      Then I should see the text "My event"
+      And I should see the text "Exhibitions"
+      And I should see the text "Short intro"
+      And I should see the text "Event description summary"
+      And I should see the text "Event description"
+      And the "Featured media legend" field should contain "Event media legend"
+      And I press "Event report"
+      And I should see the text "Event summary for report"
+      And I should see the text "Report text"
+      And I press "Registration"
+      And I should see the text "http://example.com"
+      And the "Registration capacity" field should contain "Event capacity"
+      And the "Entrance fee" field should contain "1234"
+      And I should see the text "Facebook"
+      And datetime "1 3 2020 12 30" is selected for "Start date" of "Registration date"
+      And datetime "10 3 2020 18 30" is selected for "End date" of "Registration date"
+      And I press "Online"
+      And the "Online description" field should contain "Event online description"
+      And datetime "21 2 2019 9 15" is selected for "Start date" of "Online time"
+      And datetime "21 2 2019 14 0" is selected for "End date" of "Online time"
+      And the "oe_event_online_link[0][title]" field should contain "Info site"
```

Comment:
We don't do this in OpenEuropa, Behat scenarios have to be readable, we have several methods to make this possible, check other components for examples.

Reply:
Also this is supposed to test our API steps right? Let's assert that values are set correctly by asserting that the expected values are printed out when viewing the node, not by asserting the form. oe_content can provide `default` view modes.